OMNA/registerpage.html

```diff
@@ -288,6 +288,600 @@
             </table>
         </div>
 
+        <div>
+            <h2>OMA Security</h2>
+            <table id="registries-table" class="table-bordered">
+                <thead>
+                    <tr>
+                        <th>Registry Type</th>
+                        <th>Information Requested</th>
+                        <th>Action</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td id="iso-object-identifiers">
```

Comment:
@Nathan, yes they are used to link to a particular register from "Registries.hmtl" page. I haven't done yet the PR to do the link yet.
Is this 'ID' set correctly as part of the ``<td>`` ?